packages/rest/src/server.ts

```diff
@@ -4,31 +4,35 @@ import type { Express } from 'express'
 
 import { Agent } from '@aries-framework/core'
 import { validationMetadatasToSchemas } from 'class-validator-jsonschema'
-import { static as stc } from 'express'
-import { createExpressServer, getMetadataArgsStorage, useContainer } from 'routing-controllers'
+import { createExpressServer, getMetadataArgsStorage, useContainer, useExpressServer } from 'routing-controllers'
 import { routingControllersToSpec } from 'routing-controllers-openapi'
 import * as swaggerUiExpress from 'swagger-ui-express'
 import { Container } from 'typedi'
 
 // eslint-disable-next-line @typescript-eslint/no-var-requires
 const packageJson = require('../package.json')
 
-export const setupServer = async (agent: Agent, config: ServerConfig) => {
+export const setupServer = async (agent: Agent, config: ServerConfig, app?: Express) => {
```

Comment:
In AFJ we put the `app` in the server config, maybe we can take the same approach here?

```suggestion
export const setupServer = async (agent: Agent, config: ServerConfig, app?: Express) => {
```